### PR TITLE
edited readme.md + styled card

### PR DESCRIPTION
**Pull request by lefteris-L**
[DEMO LINK](https://lefteris-l.github.io/layout_product-cards-en/) 
[TEST REPORT LINK](https://lefteris-l.github.io/layout_product-cards-en/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,20 +2,54 @@
...
+      <div class="card__stars">
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+
+        <div class="reviews">Reviews: 5</div>
```
Comment (mgrinko) [inline src/index.html:38]: Do you think this need to be an independent BEM block 
And can be used outside the card?

File: src/style.css [hunk 2]
```diff
@@ -1 +1,122 @@
...
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+/* #endregion */
+
+/* #region productInfo */
+.card__image {
```
Comment (mgrinko) [inline src/style.css:27]: card elements are also a part of card block

File: src/style.css [hunk 3]
```diff
@@ -1 +1,122 @@
...
+/* #region stars */
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.reviews {
+  color: #060b35;
```
Comment (mgrinko) [inline src/style.css:55]: please use CSS variable not to duplicate color literal

File: src/style.css [hunk 4]
```diff
@@ -1 +1,122 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-decoration: none;
+  color: #fff;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  line-height: 16px;
+  box-sizing: border-box;
```
Comment (mgrinko) [inline src/style.css:113]: please group box model rules at the top

File: src/style.css [hunk 5]
```diff
@@ -1 +1,122 @@
...
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+  line-height: 18px;
+}
+
+.card__buy {
+  height: 40px;
+  border-radius: 5px;
+  background-color: #00acdc;
```
Comment (mgrinko) [inline src/style.css:103]: just styling rules may go later

**Review by mgrinko — COMMENTED** [review]
please note my comments in you next solutions

**Review by mgrinko — APPROVED** [review]

File: src/style.css (hunk 2 above)
Comment (lefteris-L) [reply src/style.css:27]: @mgrinko Do you mean for the region group?

File: src/style.css (hunk 4 above)
Comment (lefteris-L) [reply src/style.css:113]: Thanks @mgrinko . I have noted all your comments and will try to use them in the next tasks.

**Pull request closed without merging**